app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -23,6 +23,8 @@ set -e
 <%= snippet 'remote_execution_ssh_keys' %>
 <%= snippet 'insights' -%>
 
+<%= install_packages(host_param('host_registration_packages')) unless host_param('host_registration_packages').blank? %>
+<%= snippet_if_exists('registration_linux_post') %>
```

Comment:
this will need renaming soon after the registration is renamed to host_init_config